timer.dat

```diff
@@ -0,0 +1,70 @@
+
+Host: nickrubin-macbookpro.roam.corp.google.c
```

Comment:
Remove this.

Reply:
ugh.  .dat files from psi4 seemed to be ALLOWED from the gitignore. This is changed with this PR but I missed this file. I will remove it.